jassign/jassign.py

```diff
@@ -3,20 +3,23 @@
 import argparse
 import pathlib
 
-from .to_ok import gen_views
+from to_ok import gen_views
```

Comment:
I think this change will make it impossible to run `jassign` as a command-line tool.

Reply:
If you can figure out a way to make it run both as a command-line tool and as a script, that would be great, but I didn't figure out how to make one import work for both.

Reply:
We currently use it as a CLI tool and in a make file, so I'll just revert this back.